xbmc/platform/darwin/ios/Info.plist.in

```diff
@@ -320,6 +320,7 @@
           <string>aif</string>
           <string>aiff</string>
           <string>wpl</string>
+          <string>xspf</string>
```

Comment:
Might also need to be added to UWP (and other platforms) acceptable extensions we can open

Reply:
@rechi @Paxxi maybe?

Linux doesn't need it afaik

Reply:
https://github.com/xbmc/xbmc/blob/master/tools/windows/packaging/uwp/package.appxmanifest.in